kolibri/utils/debian_check.py

```diff
@@ -13,7 +13,7 @@ def check_debian_user():
 
     with open("/etc/kolibri/username", "r") as f:
         kolibri_user = f.read().rstrip()
-    current_user = os.environ["USER"]
```

Comment:
This was necessary to run tests on a Debian system that already has Kolibri installed!